webpack/assets/javascripts/react_app/redux/API/APIHelpers.js

```diff
@@ -0,0 +1,75 @@
+import { STATUS } from '../../constants';
+
+export const formatSuccess = ({
+  data = {},
+  payload = {},
+  status = STATUS.RESOLVED,
+  error = null,
+}) => ({
+  status,
+  error,
+  ...payload,
+  ...data,
+});
+
+export const formatFailure = ({
+  error,
+  payload = {},
+  status = STATUS.ERROR,
+}) => ({
+  status,
+  error,
+  ...payload,
+});
+
+export const formatPending = ({
```

Comment:
Why do we need `formatPending`? It looks very straight forward.

Reply:
it is, though you could format pending by ID, controller or by another sorting you like.
so the store would look something like: 
```js
{
  API: {
    MY_COMPONENT_KEY: {
      7: {
        id: 7,
        title: "some-title"
        controller: "some-controller",
        status: "PENDING",
        error: null,
      },
      81: {
        id: 81,
        title: "some-title"
        controller: "some-controller",
        status: "PENDING",
        error: null,
      }
    }
  }
}
```